packages/protocol/test/common/accounts.ts

```diff
@@ -14,21 +14,6 @@ import {
 const Accounts: AccountsContract = artifacts.require('Accounts')
 const Registry: RegistryContract = artifacts.require('Registry')
 const MockValidators: MockValidatorsContract = artifacts.require('MockValidators')
-const authorizationTests: any = {}
```

Comment:
Why remove these? I.e. we should keep them to test the backwards compatability, no?

Reply:
I assume you saw that these are used further down.